lesson_07/objects/objects_app/src/test/java/com/codedifferently/lesson7/richhawkins/HouseTest.java

```diff
@@ -0,0 +1,68 @@
+package com.codedifferently.lesson7.richhawkins;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import com.codedifferently.lesson7.richhawkins.House.HouseType;
+import org.junit.jupiter.api.Test;
+
+public class HouseTest {
+
+  @Test
+  public void testSetYearBuilt() {
+    House house = new House("Pine", 234);
+
+    house.setYearBuilt(2020);
+    assertEquals(2020, house.getYearBuilt());
+
+    try {
+      house.setYearBuilt(2030);
+      fail("Exception not thrown");
```

Comment:
Use `assertThatThrownBy` (preferred) or `assertThrows` to test that an exception has been thrown.